pkg/api/openshiftcluster.go

```diff
@@ -17,10 +17,32 @@ type OpenShiftCluster struct {
 	Name       string                     `json:"name,omitempty"`
 	Type       string                     `json:"type,omitempty"`
 	Location   string                     `json:"location,omitempty"`
+	SystemData SystemData                 `json:"systemData,omitempty"`
 	Tags       map[string]string          `json:"tags,omitempty"`
 	Properties OpenShiftClusterProperties `json:"properties,omitempty"`
 }
 
+type CreatedByType string
```

Comment:
I defaulted back to original type due fact common-types generates SDK with those names. So its to align and remove confusion :/